DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -1090,12 +1091,25 @@ def onCancel(self):
 
 class GoogleCloudPlatform(object):
 
+  def findgcloud(self):
+    gcloudPath = shutil.which('gcloud')
+    if gcloudPath:
+      return gcloudPath
+
+    if sys.platform in ("linux", "darwin"):
+      # Try a second way to find gcloud path
+        shell = os.environ.get("SHELL")
+        if shell:
+          process = slicer.util.launchConsoleProcess([shell, "-c", "which gcloud"])
+          process.wait()
+          gcloudPath = process.stdout.read().strip()
+          if os.path.exists(gcloudPath):
+            return gcloudPath
+
+    logging.error(f"Unable to locate gcloud, please install the Google Cloud SDK")
```

Comment:
We should raise an exception (instead of logging a message and returning with None), because that way it is easier to display a detailed error message to the user.

Reply:
added a `RuntimeError` if `gcloud` is not found. The error message is extending to note PATH setup and the likely file (.bashrc or .zshrc) to modify if the user is using linux or macos